.pre-commit-config.yaml

```diff
@@ -1,10 +1,23 @@
 repos:
-  - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
+  - repo: local
     hooks:
-      - id: isort
-        args: ["--settings", "setup.cfg"]
-        exclude: ^tests/.*snapshots/
+      - id: ruff
+        name: ruff
+        entry: ruff check .
+        args: [ --config, pyproject.toml, --fix, --show-fixes, --exit-non-zero-on-fix ]
+        language: system
+        types_or: [python]
+        require_serial: true # run once for all files
+        pass_filenames: false
+      - id: ruff-ci
```

Comment:
this one is only used for the one CI job with the problem matcher enabled, right? does it then even need to be in pre-commit?

Reply:
Good point, it doesn't, will fix